pkgs/development/beam-modules/erlang-ls/default.nix

```diff
@@ -16,6 +16,12 @@ let
           substituteInPlace rebar.config --replace ", warnings_as_errors" ""
           '';
       });
+      json_polyfill = super.json_polyfill.overrideAttrs (_: {
+        # don't attempt to remove beam file during build, as it won't exist for erlang >= 27
+        patchPhase = ''
```

Comment:
This could be postPatch

Reply:
Doesn't seem to work – unless I'm misunderstanding something.

```diff
diff --git a/pkgs/development/beam-modules/erlang-ls/default.nix b/pkgs/development/beam-modules/erlang-ls/default.nix
index 2d6b76ca6..048de4d26 100644
--- a/pkgs/development/beam-modules/erlang-ls/default.nix
+++ b/pkgs/development/beam-modules/erlang-ls/default.nix
@@ -18,7 +18,7 @@ let
       });
       json_polyfill = super.json_polyfill.overrideAttrs (_: {
         # don't attempt to remove beam file during build, as it won't exist for erlang >= 27
-        patchPhase = ''
+        postPatchPhase = ''
           substituteInPlace rebar.config.script --replace "{erlc_compile, \"rm \\\"\$REBAR_DEPS_DIR/json_polyfill/ebin/json.beam\\\"\"}" ""
           '';
       });
```

```
$ nix build .\#beam.packages.erlang_27.erlang-ls
warning: Git tree '/Users/ehamberg/Src/nixpkgs' is dirty
error: builder for '/nix/store/54zw0cp1574blqc3r5ck5b10nza90c9x-json_polyfill-0.1.4.drv' failed with exit code 1;
       last 10 log lines:
       > no configure script, doing nothing
       > Running phase: buildPhase
       > ===> Errors loading plugin rebar3_hex. Run rebar3 with DEBUG=1 set to see errors.
       > ===> Errors loading plugin rebar3_ex_doc. Run rebar3 with DEBUG=1 set to see errors.
       > ===> Analyzing applications...
       > ===> Compiling json_polyfill
       > rm: cannot remove '/private/tmp/nix-build-json_polyfill-0.1.4.drv-0/json_polyfill-0.1.4/_build/default/lib/json_polyfill/ebin/json.beam': No such file or directory
       > ===> Hook for erlc_compile failed!
       >
       >
       For full logs, run 'nix log /nix/store/54zw0cp1574blqc3r5ck5b10nza90c9x-json_polyfill-0.1.4.drv'.
```

Reply:
The name is just postPatch

Reply:
Aaah! That worked. Fixed!